### ISSUE TITLE
Add "Forgot Password" function

### ISSUE THREAD

**Issue by ETWang1991** [IceWhaleTech/CasaOS#725]
mud: The best thing I've done is to take a snapshot before creating the initial root account. So even if I lost it it can be easy to reconfigure. However, I haven't got to resetting the root pass. Hope there is one

**Comment by OptimusGREEN** [comment]
yeah I currently can't log in. It appears to have reset the login to admin/casaos defaults but i cant choose the user, its stuck at my previous user

**Comment by whitemankpi** [comment]
same problem. Somehow cant log in and cant logout and cant use another account\user
